chef/cookbooks/neutron/templates/default/ovs_neutron_plugin.ini.erb

```diff
@@ -125,8 +125,12 @@ bridge_mappings = physnet1:br-fixed
 # Example: tunnel_types = gre
 # Example: tunnel_types = vxlan
 # Example: tunnel_types = vxlan, gre
-<% if @ml2_type_drivers.include?("gre") -%>
+<% if @ml2_type_drivers.include?("gre") && @ml2_type_drivers.include?("vxlan") -%>
```

Comment:
I feel like this should be doable using `case ... when ... end` but maybe it isn't :-/